src/clj_kondo/impl/analyzer.clj

```diff
@@ -1663,7 +1663,7 @@
         arg-count (if (and transducer-eligable?
                            (zero? arg-count)) ;; transducer
                     (if (and (= 'clojure.core hof-ns-name)
-                             (= 'map hof-resolved-name))
+                             (#{'map 'mapcat} hof-resolved-name))
```

Comment:
Although probably not so important, I usually abstain from this style to keep performance optimal and write out:

```
(or (= 'map ...) (= 'mapcat ...))
```